src/Core/Tool/Authorizer/UserAuthorizer.php

```diff
@@ -43,6 +43,12 @@ protected function getAllPrivs()
 	/* Authorizer */
 	public function isAllowed(Entity $entity, $privilege)
 	{
+		// Users cannot create 'owner' role
+		if (in_array($privilege, ['register', 'create', 'update'])
+			and $entity->role === 'owner') {
```

Comment:
Yeah. This is going to be a problem to implement if admin is not allowed to do that...